### ISSUE TITLE
CI job should still run if only Cargo.toml changes

### ISSUE THREAD

**Issue by obycode** [stx-labs/clarinet#462]

**Comment by csgui** [comment]
Hello @obycode !

I would like to contribute with this issue. Could you please provide more info on that? When changes to `Cargo.toml` aren't triggering the CI? Is on a pull request creation? Or on a push to branches?

Thanks!

**Comment by obycode** [comment]
Thanks @csgui. There have been some changes to the CI configuration since this issue was posted, so it may not be relevant anymore. Perhaps we should first start by opening a PR where only the Cargo.toml is changed and then check if the CI jobs run. When I created this issue, there had been a PR where they didn't run in that scenario, but I have a feeling it has since been fixed through unrelated changes.

**Comment by csgui** [comment]
> Perhaps we should first start by opening a PR where only the Cargo.toml is changed and then check if the CI jobs run.

Hi @obycode,

I was checking the [Cargo.toml](https://github.com/hirosystems/clarinet/blob/develop/Cargo.toml) and notice that the `clarity-repl` component is missing from the members list.

We can add that to the list and check if the CI will run. What you think?

**Comment by obycode** [comment]
Yes, that sounds good to me. @lgalabru - is that left out intentionally for any reason?
